docker/nginx/libs/skynet/account.lua

```diff
@@ -1,13 +1,15 @@
 local _M = {}
 
 -- fallback - remember to keep those updated
-local anon_limits = { ["tierName"] = "anonymous", ["upload"] = 655360, ["download"] = 655360, ["maxUploadSize"] = 1073741824, ["registry"] = 250 }
+local anon_limits = { ["tierID"] = 0, ["tierName"] = "anonymous", ["upload"] = 655360, ["download"] = 655360, ["maxUploadSize"] = 1073741824, ["registry"] = 250 }
```

Comment:
Wouldn't it be better to do:

```suggestion
local anon_limits = { ["tierID"] = tier_id_anonymous, ["tierName"] = "anonymous", ["upload"] = 655360, ["download"] = 655360, ["maxUploadSize"] = 1073741824, ["registry"] = 250 }
```

? (if possible with Lua ¯\_(ツ)_/¯)

Reply:
addressed in https://github.com/SkynetLabs/skynet-webportal/pull/1789/commits/30e8c886b93313ad0d205b0bed25682e98f8d2e0